AnnoyancesFilter/sections/tweaks.txt

```diff
@@ -426,8 +426,8 @@ safelink.rezkozpatch.xyz#%#!function(){if(-1<window.location.href.indexOf(".html
 hikarinoakari.com#%#//scriptlet("adjust-setInterval", "count", "", "0.02")
 ! fcc.lc - skip timers
 short.fc-lc.com#%#AG_onLoad(function() { setTimeout(function() { var el = document.querySelector('#submitbtn'); if(el) { el.click(); } }, 300); });
-articlix.com,fcc.lc#%#AG_onLoad(function() { setTimeout(function() { var el = document.querySelector("#invisibleCaptchaShortlink"); if(el) { el.classList.remove("disabled"); el.removeAttribute("disabled"); el.click(); } }, 300); });
-articlix.com,fcc.lc#%#AG_onLoad(function(){setTimeout(function(){var a=document.querySelector("#surl"),c=function(){var d=jQuery.ajax({url:domain,method:"POST",dataType:"json",data:jQuery("#form12").serialize()});setTimeout(function(){try{var b=d.responseText.split(/https?/)[1];b=b.split('"')[0];a.href="https"+b;a.text="Get Link";a.classList.add("btn-success");a.click()}catch(e){console.log("Failed, try again..."),c()}},1E3)};a&&(a.classList.remove("disabled"),c())},500)});
+fclcc.com,articlix.com,fcc.lc#%#AG_onLoad(function() { setTimeout(function() { var el = document.querySelector("#invisibleCaptchaShortlink"); if(el) { el.classList.remove("disabled"); el.removeAttribute("disabled"); el.click(); } }, 300); });
```

Comment:
It seems that they fixed their captcha and it can't be skipped now.
<details><summary>Screenshot</summary>

![image](https://user-images.githubusercontent.com/29142494/115010474-c2d1eb00-9ead-11eb-9c5f-d7c3a69528d3.png)

</details>

So I think that we should comment this rule (add `!` before it).

Reply:
It's impossible to fix now, from my view. Isn't it?

Reply:
Skipping captcha is rather impossible, but it seems that timer still can be skipped, like before.

Reply:
Yeah, I have seen.